db/migrate/20220204211419_add_details_to_posts.rb

```diff
@@ -0,0 +1,10 @@
+class AddDetailsToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_column :posts, :title, :string
+    add_column :posts, :text, :string
```

Comment:
Please change the data type of the `text` attribute to `text` as it will be holding a few lines of details for the post but the `string` data type is for attributes such as `username`, and `password` etc.